w-near/Cargo.toml

```diff
@@ -5,14 +5,18 @@ authors = ["Near Inc <hello@nearprotocol.com>", "Vincent <vincent@decentralisede
 edition = "2018"
 
 [lib]
-crate-type = ["cdylib", "rlib"]
+crate-type = ["cdylib"]
 
 [dependencies]
-near-sdk = "3.1.0"
-near-contract-standards = "3.1.0"
+near-sdk = "4.0.0-pre.6"
+near-contract-standards = "4.0.0-pre.6"
```

Comment:
Note to whoever manages this contract that the version is being updated.

Prob worth using pre.7 to get the events added in with the ft?

Reply:
Yeah I do not see a reason not to upgrade. Done!